SPECS/rust/rust.spec

```diff
@@ -33,6 +33,7 @@ BuildRequires:  git
 BuildRequires:  glibc
 BuildRequires:  ninja-build
 BuildRequires:  python3
+BuildRequires:  libgit2-devel
```

Comment:
Although this is a welcome change, I believe we have cached all the deps as per line `21`.
I think, even though we add a BR on `libgit2-devel` I am not sure would we even consume it.
The build for `rustc` consumes the prep-prepared cache.
I believe we should maybe update that cache or altogether bump the rust version.